tests/validate.test.ts

```diff
@@ -0,0 +1,91 @@
+import { safeDump } from 'js-yaml'
+import { AddressInfo, Server } from 'net'
+import request from 'request-promise-native'
```

Comment:
Why not use fetch for everything?

Reply:
It's only part of the dev environment, I am just used to using it for tests.